src/NumericalAlgorithms/RootFinding/NewtonRaphson.hpp

```diff
@@ -22,6 +22,10 @@ namespace RootFinder {
  *
  * \snippet Test_NewtonRaphson.cpp double_newton_raphson_root_find
```

Comment:
Just drop this commit. People can read the boost documentation.

Reply:
I couldn't find anything stating this in the Boost docs, but that's probably because it doesn't seem to be true.  If I modify the test to put the lower bound `1e-10` above the correct result (requesting 8 digits), I do not get the correct answer to 8 digits.

So yes, please drop this.

Reply:
I had modified the test and determined that lowering the upper bound did not break the algorithm, but then I incorrectly generalized. I agree that raising the lower bound _does_ break the algorithm. ¯\\\_(ツ)\_/¯

I will drop this.